docs/sdk/server-side-sdks/dotnet-local/dotnet-local-usage.md

```diff
@@ -77,12 +74,12 @@ client.FlushEvents();
 
 ## Set Client Custom Data
 
-To assist with segmentation and bucketing you can set a custom data dictionary that will be used for all variable and feature evaluations. User specific custom data will override client custom data.
+To assist with segmentation and bucketing you can set a custom data dictionary that will be used for all variable and feature evaluations. DevCycleUser specific custom data will override client custom data.
```

Comment:
This one might be better contextualized as the concept of a user - rather than the name of the class itself?